test-result-summary-client/src/Build/Summary/ResultSummary.jsx

```diff
@@ -127,10 +127,6 @@ export default class ResultSummary extends Component {
                 <TestBreadcrumb buildId={parentId} />
                 <Overview id={parentId} parentBuildInfo={parentBuildInfo} summary={summary} failedSdkBuilds={failedSdkBuilds}/>
                 <Divider />
-                <div style={{ display: "flex" }}>
-                    <PieChart buildMap={buildMap} dataKey="passed" selectedPlatforms={selectedPlatforms} selectedJdkVersions={selectedJdkVersions} selectedJdkImpls={selectedJdkImpls} hcvalues={hcvalues} name="Passed Tests" showInLegend={true} dataLabels={true} />
-                    <PieChart buildMap={buildMap} dataKey="failed" selectedPlatforms={selectedPlatforms} selectedJdkVersions={selectedJdkVersions} selectedJdkImpls={selectedJdkImpls} hcvalues={hcvalues} name="Failed Tests" showInLegend={true} dataLabels={true} />
-                </div>
                 <Divider />
```

Comment:
Could you also remove this `<Divider />`? Thanks

Reply:
done :)